.github/workflows/test-phpunit.yml

```diff
@@ -55,5 +59,11 @@ jobs:
       - name: Install dependencies
         run: composer update --ansi
 
+      - name: Downgrade src
+        run: php bin/transform-source ${{ env.TARGET_PHP_VERSION_ID }}
+
+      - name: Run lint on src
+        run: vendor/bin/parallel-lint src --colors --show-deprecated
```

Comment:
It seems you do not check on PHP 7.4. Is it intentional?

Reply:
That's my original intention. But `composer update` fails: https://github.com/CodeIgniter/phpstan-codeigniter/actions/runs/6873219405/job/18692938215

If I will reinstall PHP for PHP 7.4, I will first remove PHPUnit 10 to run lint then reinstall again PHP 8.1 to run tests. It's tedious. Unless you have another flow in mind?

Reply:
Okay, frankly, I feel it is tedious to downgrade.
I want to drop PHP 7.4 and 8.0!

Reply:
This downgrading is just a hack for now while v4.5 is not yet out.